### DISCUSSION TITLE
Change page title

### DISCUSSION THREAD

**Discussion by mmelendezvega** [opendatacam/opendatacam#604] [Q&A] [answered]
Is it possible to change the default "OpenDataCam" page title?

![odc](https://github.com/opendatacam/opendatacam/assets/82361524/3ce1c6d1-bcbd-4ab4-9959-c63bcac24d18)

**Comment by vsaw** [comment, marked as answer]
Of course it is. It's defined in https://github.com/opendatacam/opendatacam/blob/master/components/shared/Layout.js. You would have to run ODC from source or rebuild the docker image though

**Comment by mmelendezvega** [comment]
Thanks!
